content/api/webhooks.apib

```diff
@@ -138,61 +187,115 @@ As described in "Webhooks Object Properties", webhook creation entails a test PO
             ]
         }
 
-## Retrieve [/webhooks/{id}{?timezone}]
+### Validate a Webhook [POST /webhooks/{id}/validate]
 
-+ Model
+The validation sends an example message event batch from the Webhooks API to the
```

Comment:
Sends a sample event batch to the target URL, validates that the target responds with HTTP 200, and returns information on the response received from the target.

Sample Batch
    ```
        [
            {
              "msys": { }
            }
        ]
    ```